### PR TITLE
Add `-fetchRepositoryReadme:reference:` method to `OCTClient+Repositorie...

### PR DESCRIPTION
**Pull request by leichunfeng**
...s` category.

### CODE REVIEW

File: OctoKit/OCTClient+Repositories.h [hunk 1]
```diff
@@ -71,6 +71,15 @@
 // Returns a signal which will send zero or one OCTContent.
 - (RACSignal *)fetchRepositoryReadme:(OCTRepository *)repository;
 
+// Fetches the readme of a `repository` by the given `reference`.
+//
+// repository - The repository for which the readme should be fetched.
+// reference  - The name of the commit, branch or tag, may be `nil` in which
+//				case it defaults to the default repo branch.
```
Comment (jspahrsummers) [inline OctoKit/OCTClient+Repositories.h]: Can you please fix the indentation here? It looks like it may be using tabs instead of spaces.

**Comment by jspahrsummers** [comment]
Thanks! Just one note. :pencil2:

**Comment by leichunfeng** [comment]
Of course, I have fixed it.

**Comment by jspahrsummers** [comment]
:shipit:

**Pull request merged**